platform/ext/target/musca_a/plat_test.c

```diff
@@ -96,3 +98,4 @@ void tfm_plat_test_non_secure_timer_stop(void)
     timer_cmsdk_disable_interrupt(&CMSDK_TIMER1_DEV_NS);
     timer_cmsdk_clear_interrupt(&CMSDK_TIMER1_DEV_NS);
 }
+#endif //DOMAIN_NS == 0
```

Comment:
TF-M style seems not to put `//BLAH` after their `#endif` directives. This ifdef is small enough we probably don't need any comment here.

Reply:
Looks like it is file specific. The instances I saw https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git/tree/secure_fw/spm/init/tfm_core.c#n162 seems to add `/* BLAH */` after `#endif`. Also there are other `#if`s without comments. I like adding the comment which improves code readability. But I'll change it to classic commenting style.